coalib/__init__.py

```diff
@@ -21,6 +71,10 @@ def get_version():
 __version__ = VERSION
 
 
+COMMIT = get_commit(COALIB_DIR, COMMIT_FILE)
```

Comment:
this is a large startup overhead for the properly installed package, which is not a git repo.
The overhead for the properly installed package should be as **small** as possible.  No `chdir`; no `subprocess`.
The overhead for the development package, in a git repo, can be large, because it is not the production version.